src/js/grapl-cdk/lib/grapl-cdk-stack.ts

```diff
@@ -828,10 +848,13 @@ export class GraplCdkStack extends cdk.Stack {
 
         const dgraphSwarmCluster = new DGraphSwarmCluster(
             this,
-            'dgraph-swarm-cluster',
+            'swarm',
             {
                 prefix: this.prefix,
                 vpc: grapl_vpc,
+                version: props.version || 'latest',
```

Comment:
thanks to the above
`export const graplVersion`
i don't think we need to do this default anymore; and it looks like `.version` is not `?` anyway

Reply:
fixed